ush/radmon_diag_ck.sh

```diff
@@ -71,23 +71,23 @@ echo "--> radmon_diag_ck.sh"
    #---------------------------------------------
    #  get list of diag files in the radstat file 
    #
-   radstat_contents=`tar -tf ${radstat_file} | grep '_ges' |  
+   radstat_contents=`tar -tf "${radstat_file}" | grep '_ges' |
 		gawk -F"diag_" '{print $2}' | 
 		gawk -F"_ges" '{print $1}'`
 
   
    #---------------------------------------------
    #  load contents of satype_file into an array
    #
-   satype_contents=`cat ${satype_file}`
+   satype_contents=`cat "${satype_file}"`
```

Comment:
Kill the evil backticks.
```suggestion
   satype_contents=$(cat "${satype_file}")
```